container_solaris_test.go

```diff
@@ -0,0 +1,62 @@
+// +build !linux
+
+package containerd
+
+import (
+	"syscall"
+	"testing"
+)
+
+func TestContainerUpdate(t *testing.T) {
```

Comment:
This doesn't do any update of the container resources at all. If that's not supported on Solaris, no point on having a test